specification/maps/data-plane/Common/preview/1.0/common.json

```diff
@@ -1,712 +1,712 @@
+        "in": "query",
+        "description": "Y coordinate of the tile on zoom grid. Value must be in the range [0, 2<sup>`zoom`</sup> -1].\n\nPlease see [Zoom Levels and Tile Grid](https://docs.microsoft.com/en-us/azure/location-based-services/zoom-levels-and-tile-grid) for details.",
+        "required": true,
+        "type": "integer",
+        "format": "int32",
+        "x-ms-parameter-location": "method"
       }
     },
-    "202AsyncV2": {
-      "description": "Request Accepted: The request has been accepted for processing. Please use the URL in the Operation-Location Header to obtain status.",
-      "headers": {
-        "Operation-Location": {
-          "type": "string",
-          "description": "New URL to check for the results of the long running process."
+    "responses": {
+      "200AsyncV2": {
+        "description": "The operation is running or complete. If the operation was successful, use the Resource-Location header to obtain the path to the result.",
+        "schema": {
+          "$ref": "#/definitions/LongRunningOperationResult"
+        },
+        "headers": {
+          "Resource-Location": {
+            "type": "string",
+            "description": "If successful, a URI where details on the newly created resource can be found."
+          }
         }
-      }
-    },
-    "408": {
-      "description": "Timeout: The request took longer to finish than the maximum amount of time defined in the api.",
-      "schema": {
-        "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorResponse"
       },
-      "x-ms-error-response": true
-    },
-    "409": {
-      "description": "The request could not be completed due to a conflict with the current state of the resource. Likely caused by a parallel update to the same resource.",
-      "schema": {
-        "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorResponse"
+      "202Async": {
+        "description": "**Supported only for async request.**\nRequest Accepted: The request has been accepted for processing. Please use the URL in the Location Header to retry or access the results.",
+        "headers": {
+          "Location": {
+            "type": "string",
+            "description": "New URL to check for the results of the long running process."
+          }
+        }
       },
-      "x-ms-error-response": true
-    },
-    "409StorageLimit": {
-      "description": "The [data storage limit](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#azure-maps-limits) is reached on the Azure Maps account. You can always use the [Data Delete API](https://docs.microsoft.com/en-us/rest/api/maps/data/deletepreview) to delete old/unused content and create space for new uploads.",
-      "schema": {
-        "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorResponse"
+      "202AsyncV2": {
```

Comment:
There should not be a V1 and V2 async in here. It looks like 202Async is only referenced by v1 APIs and 202AsyncV2 is referenced only by V2 APIs. The way you accomplish this is that there should be a v2 Common which defines the common references for V2. In this instance, the 202Async goes away since it's only used by V1 APIs and "202AsyncV2" becomes "202Async". 

This also means that all references from V2 APIs need to point to `2.0/Common` whereas right now everything is pointing to v1.